app/src/main/java/io/gnosis/safe/ui/settings/safe/SafeSettingsFragment.kt

```diff
@@ -114,51 +114,53 @@ class SafeSettingsFragment : BaseViewBindingFragment<FragmentSettingsSafeBinding
             localName.name = safe?.localName
             threshold.name = getString(R.string.safe_settings_confirmations_required, safeInfo?.threshold, safeInfo?.owners?.size)
             ownersContainer.removeAllViews()
-            safeInfo?.owners?.forEach { owner -> ownersContainer.addView(ownerView(safe!!.chain, owner, localOwners)) }
+            safeInfo?.owners?.dropLast(1)?.forEach { owner -> ownersContainer.addView(ownerView(safe!!.chain, owner, localOwners)) }
```

Comment:
why is this needed? Sequence seems to be same as before, with the previous version using one forEach loop.

Reply:
I found something more readable without the index. 

```
safeInfo?.owners?.forEach() { owner ->
    ownersContainer.addView(ownerView(safe!!.chain, owner, localOwners, owner != safeInfo.owners.last()))
}
```